src/Umbraco.Web.UI.Client/src/packages/search/examine-management-dashboard/views/section-view-examine-searchers.ts

```diff
@@ -158,13 +179,19 @@ export class UmbDashboardExamineSearcherElement extends UmbLitElement {
 		}
 	}
 
+	#onPageChange(event: UUIPaginationEvent) {
+		this.#paginationManager.setCurrentPageNumber(event.target?.current);
+		this._onSearch();
+	}
+
 	private renderSearchResults() {
 		if (this._searchLoading) return html`<uui-loader></uui-loader>`;
 		if (!this._searchResults) return nothing;
 		if (!this._searchResults.length) {
 			return html`<p>${this.localize.term('examineManagement_noResults')}</p>`;
 		}
-		return html`<div class="table-container">
+		return html`<div>${this.localize.term('examineManagement_totalResults')}: ${this._total}</div>
```

Comment:
Could we look to render something more like what we have on Umbraco 13 here, rather than just the total?  We already have a localization key of `searchResultsFound`.  It has placeholders for the total items, number of pages etc., but should be turned into a function where these values are passed in (like some others are, e.g. `confirmTrash`).

Reply:
If you do this, you can remove the new localization key you have added.